io.openems.edge.timeofusetariff.entsoe/test/io/openems/edge/timeofusetariff/entsoe/ExchangeRateApiTest.java

```diff
@@ -16,199 +15,39 @@ public class ExchangeRateApiTest {
-					"MDL": 19.474446,
-					"MGA": 4984.515277,
-					"MKD": 61.641818,
-					"MMK": 2330.490351,
-					"MNT": 3903.398698,
-					"MOP": 8.922849,
-					"MRU": 37.929989,
-					"MUR": 50.325954,
-					"MVR": 17.027553,
-					"MWK": 1168.023721,
-					"MXN": 18.676827,
-					"MYR": 5.024866,
-					"MZN": 70.714101,
-					"NAD": 19.579235,
-					"NGN": 874.632255,
-					"NIO": 40.574721,
-					"NOK": 11.166941,
-					"NPR": 145.603541,
-					"NZD": 1.775313,
-					"OMR": 0.427697,
-					"PAB": 1.110007,
-					"PEN": 3.990703,
-					"PGK": 3.980789,
-					"PHP": 60.551875,
-					"PKR": 318.444017,
-					"PLN": 4.41964,
-					"PYG": 8080.356808,
-					"QAR": 4.039405,
-					"RON": 4.923793,
-					"RSD": 117.135798,
-					"RUB": 99.859233,
-					"RWF": 1302.494419,
-					"SAR": 4.160461,
-					"SBD": 9.298905,
-					"SCR": 14.821421,
-					"SDG": 667.205497,
-					"SEK": 11.504976,
-					"SGD": 1.469025,
-					"SHP": 0.857181,
-					"SLL": 19594.637867,
-					"SOS": 631.983688,
-					"SRD": 42.561687,
-					"SSP": 144.489173,
-					"STD": 25317.170823,
-					"STN": 24.847059,
-					"SVC": 9.710443,
-					"SYP": 2786.987107,
-					"SZL": 19.625727,
-					"THB": 37.81308,
-					"TJS": 12.130966,
-					"TMT": 3.882521,
-					"TND": 3.428976,
-					"TOP": 2.599614,
-					"TRY": 29.846988,
-					"TTD": 7.539697,
-					"TWD": 34.634021,
-					"TZS": 2717.626941,
-					"UAH": 40.986807,
-					"UGX": 4049.121605,
-					"USD": 1.109987,
-					"UYU": 42.02786,
-					"UZS": 12905.949812,
-					"VES": 32.209006,
-					"VND": 26260.304612,
-					"VUV": 131.976192,
-					"WST": 3.024415,
-					"XAF": 655.638919,
-					"XAG": 0.044704,
-					"XAU": 0.001581,
-					"XCD": 2.997771,
-					"XDR": 0.825364,
-					"XOF": 655.639063,
-					"XPD": 0.001494,
-					"XPF": 119.274762,
-					"XPT": 0.001957,
-					"YER": 277.641934,
-					"ZAR": 19.492845,
-					"ZMW": 21.436533,
-					"ZWL": 357.173757
+				"quotes": {
+					"EURAED": 3.894874,
+					"EURAFN": 79.984782,
+			     	"EURALL": 105.884804,
+			     	"EURAMD": 421.63859,
+				    "EURANG": 1.911328,
+				    "EURAOA": 877.465407,
+				    "EURSCR": 13.852121,
+				    "EURSDG": 637.697611,
+				    "EURSEK": 11.562427,
+				    "EURSGD": 1.445276,
+				    "EURSHP": 1.290222,
+				    "EURZWL": 341.443067
 				}
 			}
 			""";
 
 	@Test
 	@Ignore
 	public void testExchangeRateApi() throws IOException {
-		ExchangeRateApi.getExchangeRates();
+		// Enter personal access key and remove '@Ignore' tag to test it.
+		var accessKey = "";
+		ExchangeRateApi.getExchangeRates(accessKey);
 	}
 
 	@Test
 	public void testExchangeRateParser() throws OpenemsNamedException {
 
-		var currency = Currency.EUR;
+		var currency = Currency.SEK;
 		var response = Utils.exchangeRateParser(EXCHANGE_DATA, currency);
 
-		assertTrue(response == 1.0);
-
-		currency = Currency.SEK;
-		response = Utils.exchangeRateParser(EXCHANGE_DATA, currency);
-
-		assertFalse(response == 1.0);
+		assertTrue(response == 11.562427);
```

Comment:
Float or Double values should never be compared like this. Use `assertEquals(11.562427, rate, 0.0001);` instead.